packages/coachmark/README.md

```diff
@@ -38,9 +39,28 @@ import { Coachmark } from '@spectrum-web-components/coachmark';
 </sp-coachmark>
 ```
 
-## Using Action Menu
+#### Multi-step Tour
 
-Coach marks can include an `<sp-action-menu>`, which appears at the top right of the coach mark. The `<sp-action-menu>` should only include ways to interact with the coach mark tour as a whole, with options like “Skip tour” or “Restart tour.”
+Coachmarks can be part of a multi-step tour with navigation buttons:
+
+```html
+<sp-coachmark
```

Comment:
Thanks for adding this. Can we make an example showing how navigation works between multiple steps?